lib/shanty/plugins/shantyfile_plugin.rb

```diff
@@ -6,6 +6,7 @@ module Plugins
     # Public: Plugin for finding all directories marked with a Shantyfile.
     class ShantyfilePlugin < Plugin
       provides_projects :shantyfile_projects
+      description 'Parses Shantyfile in project directories'
```

Comment:
This misses one other important aspect of the `Shantyfile` which is to force that directory to become a project:

> Discovers projects with a Shantyfile and executes any instructions within it